src/tee.c

```diff
@@ -162,7 +162,7 @@ tee_version(struct tcb *const tcp, const kernel_ulong_t arg)
 	struct tee_ioctl_version_data version;
 
 	if (entering(tcp)) {
-		tprint_arg_next();
+		tprints_arg_next_name("argp");
 		return 0;
 	}
```

Comment:
Let's move this `tprints_arg_next_name("argp");` invocation after the `entering` branch.